RyuSocks/RyuSocks.csproj

```diff
@@ -37,7 +37,7 @@ The full changelog is available at: https://github.com/TSRBerry/RyuSOCKS/blob/ma
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="NetCoreServer" Version="8.0.7" />
+        <PackageReference Include="NetCoreServer" />
```

Comment:
Is the version not required anymore?
I assume this is due to not needing a specific older NetCoreServer version and just using the newest one.

Reply:
The versions are now specified in `Directory.Packages.props`